sensor_msgs/msg/NavSatStatus.msg

```diff
@@ -4,12 +4,13 @@
 # type and the last time differential corrections were received.  A
 # fix is valid when status >= STATUS_FIX.
 
+int8 STATUS_UNKNOWN = -2        # status is not yet set
 int8 STATUS_NO_FIX =  -1        # unable to fix position
 int8 STATUS_FIX =      0        # unaugmented fix
 int8 STATUS_SBAS_FIX = 1        # with satellite-based augmentation
 int8 STATUS_GBAS_FIX = 2        # with ground-based augmentation
 
-int8 status
+int8 status = STATUS_UNKNOWN
```

Comment:
And for default value syntax you don't use the `=` symbol. And I don't think that you can use the constants in the default value, though I'm not sure on that. You may need to convert it to the numeric value.

Reply:
Converting to numeric and removing the `=` is complete